tests/merge_sorted_lists_test.py

```diff
@@ -1,2 +1,38 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
+
```

Comment:
``` python
import random
```

Reply:
I don't think that's a good idea. But thank you for the suggestion.